lib/compat/wordpress-5.9/get-global-styles-and-settings.php

```diff
@@ -158,7 +158,7 @@ function wp_get_global_styles_svg_filters() {
 
 		$supports_theme_json = WP_Theme_JSON_Resolver_Gutenberg::theme_has_support();
 
-		$origins = array( 'default', 'theme' );
```

Comment:
Hey, the code in the folder `lib/compat/wordpress-5.9` is expected to be part of WordPress 5.9? Is this being ported to WordPress core? I presume it isn't, so we'd need to revert this PR and look at different ways to do this.

Reply:
Revert here: https://github.com/WordPress/gutenberg/pull/38150

Reply:
Why can't we just port to core instead of reverting and working around it here?

Reply:
Yeah, if the change was part of 5.9, this would have been fine. Though when this landed only the 5.9 leads could approve PRs for core and that would need to happen before.

In general, code in a `lib/compat/wordpress-X.Y` folder needs to be in sync with that same WordPress X.Y release.